docs/great-houses/house-of-defi.md

```diff
@@ -1,42 +1,44 @@
+
```

Comment:
Hey @DeFiDude thanks for the update! 🍻 
Could you add a return after the frontmatter block (the `---` on line 3) and remove the empty line on line 1 please? It'll fix the missing icon in the sidebar and the broken formatting in the page title. 👇🏻 
![image](https://user-images.githubusercontent.com/780350/194712058-83c57e32-8234-419d-88fb-adfe61a71c28.png)
![image](https://user-images.githubusercontent.com/780350/194712407-52ad6758-de40-4f15-8be6-40684a043412.png)